### PR TITLE
Md 8033 v1 : Created color picker component

### PR DESCRIPTION
**Pull request by abhishekraj1007**
jira : https://parspec.atlassian.net/browse/MD-8033

### CODE REVIEW

File: src/components/ColorPicker/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+
+import { SketchPicker, SketchPickerProps, ColorResult } from 'react-color';
+import { Box } from '../Box';
+import { CustomPopper, Fade } from '../Popper';
+import { ClickAwayListener } from '@mui/material';
+
```
Comment (atul2103) [inline src/components/ColorPicker/index.tsx:7]: move mui import above the box and popper imports

File: src/components/Popper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Popper, { PopperProps } from '@mui/material/Popper';
+
+import { Fade } from '@mui/material';
+
+export const CustomPopper: React.FC<PopperProps> = ({ children, ...props }) => {
+    return <Popper {...props}>{children}</Popper>;
+};
+
+export { Fade };
```
Comment (atul2103) [inline src/components/Popper/index.tsx:9]: why Fade is exported.

File: src/components/Popper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import Popper, { PopperProps } from '@mui/material/Popper';
+
+import { Fade } from '@mui/material';
+
+export const CustomPopper: React.FC<PopperProps> = ({ children, ...props }) => {
```
Comment (atul2103) [inline src/components/Popper/index.tsx:5]: why this is not default export.

File: src/components/ColorPicker/index.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    return (
+        <>
+            <Box sx={{ cursor: 'pointer' }} onClick={handleClick} width={24} height={24} borderRadius={100} bgcolor={color}></Box>
+            {open && (
+                <ClickAwayListener onClickAway={clickAwayHandler}>
+                    <Box>
+                        <CustomPopper placement="auto" open={open} anchorEl={anchorEl} transition>
+                            {({ TransitionProps }) => (
+                                <Fade {...TransitionProps} timeout={350}>
+                                    <Box sx={{ m: 2 }}>
```
Comment (atul2103) [inline src/components/ColorPicker/index.tsx:47]: can we import fade directly here from MUI

Comment (atul2103) [inline src/components/ColorPicker/index.tsx:47]: we can pass m={2} directly without sx

File: src/components/ColorPicker/index.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+interface ITransitionsColorPicker {
+    color: string;
+    onChange: (param: ColorResult) => void;
+    onClickAway?: () => void;
+}
+
+export { ColorResult };
+
+export const TransitionsColorPicker: React.FC<ITransitionsColorPicker> = ({ color, onChange, onClickAway = () => {} }) => {
```
Comment (atul2103) [inline src/components/ColorPicker/index.tsx:20]: can we export it as default ?

**Review by atul2103 — CHANGES_REQUESTED** [review]

File: src/components/ColorPicker/index.tsx (hunk 1 above)
Comment (abhishekraj1007) [reply src/components/ColorPicker/index.tsx:7]: done

File: src/components/Popper/index.tsx (hunk 2 above)
Comment (abhishekraj1007) [reply src/components/Popper/index.tsx:9]: done

File: src/components/Popper/index.tsx (hunk 3 above)
Comment (abhishekraj1007) [reply src/components/Popper/index.tsx:5]: done

File: src/components/ColorPicker/index.tsx (hunk 4 above)
Comment (abhishekraj1007) [reply src/components/ColorPicker/index.tsx:47]: we can

File: src/components/ColorPicker/index.tsx (hunk 5 above)
Comment (abhishekraj1007) [reply src/components/ColorPicker/index.tsx:20]: let it be named export since there are two variations of colorPicker component

**Review by atul2103 — APPROVED** [review]

**Review by sunnyparspec21 — APPROVED** [review]

File: src/components/Popper/index.tsx [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import Popper, { PopperProps } from '@mui/material/Popper';
+
+export default function CustomPopper({ children, ...props }: PopperProps) {
```
Comment (abhijit-parspec) [inline src/components/Popper/index.tsx:3]: @abhishekraj1007 , keep the name as `Popper` only, no need to have the name as CustomPopper

Comment (abhishekraj1007) [reply src/components/Popper/index.tsx:3]: Done

**Review by abhijit-parspec — APPROVED** [review]

**Pull request merged**